usaspending_api/database_scripts/etl/award_delta_view.sql

```diff
@@ -81,7 +81,10 @@ SELECT
   vw_award_search.naics_code,
   vw_award_search.naics_description,
   TREASURY_ACCT.tas_paths,
-  TREASURY_ACCT.tas_components
+  TREASURY_ACCT.tas_components,
+  DEFC.defc as disaster_emergency_fund_codes,
+  DEFC.total_covid_obligations,
```

Comment:
Is it possible to store the obligation and outlay per DEFC in the index? That will make it useful for the COVID-19 page.

Reply:
@tony-sappe 
I think the effort to do this would be a lot. It would be akin to what we tried to do with the TAS records at one point in time -- making them _nested objects_ (sub json docs to the main award doc).

An alternative would be to bake those into an array of json strings like we do with TAS now (we had to get away from nested objects for indexing performance reasons). But that's usually only good for _search_ I think. And the DEFC amounts are numeric values we'd want to use for aggregation. 

There also is nothing yet in the _current_ mocks for Jul 30th that require further subdivision of award data by DEFC. Eventually if a DEFC filter comes along, it would drive something like this. 
a) We'd either need to get it into awards ES index (could be dirty and compromise the index for other uses)
b) Switch those APIs from using the awards ES index to using postgres tables instead
c) make a new account index that brings over award-specific metadata at index-time by joining `financial_accounts_by_awards` to `awards`